aioredisqueue/queue.py

```diff
@@ -4,15 +4,36 @@
 
 
 class Queue(object):
-
-    def __init__(self, redis, key_prefix='aioredisqueue_', *,
```

Comment:
Looks like you made keyword-only arguments position dependent.

Reply:
It did not conform to my `autopep`, it didn't contain this rule. Will fix it

Reply:
This is out of scope of autopep (or even and better - `black`). But if you have a lot of arguments and some of them may change in the future, allowing passing arguments as positional could be harmful.